Gruntfile.js

```diff
@@ -5,9 +5,22 @@ module.exports = function(grunt)
   grunt.loadNpmTasks('grunt-contrib-copy');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-less');
+  grunt.loadNpmTasks('grunt-eslint');
 
   // load the configuration
   grunt.initConfig({
+    eslint: {
+      options: {
+        configFile: '.eslintrc.json'
+      },
+      target: [
```

Comment:
Is it possible to make it target all js files like this? `app/**/*.js`

Reply:
Yes, but I don't know if we actually want it to because when I tried that it freaked out about all the jQuery stuff in the app/public directory. But maybe I should include it and then just tell it $ is a global? Although $ = undef was not the only issue...

Reply:
I can see the benefit in using a wildcard here, since it'll catch new files as they're added. I believe `grunt` has a way to exclude files. Something like:

```
target: [
  'app/**/*.js',
  '!app/public'
]
```

can we give that a shot and see if that works?